Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
All I can tell you is that Rubocop flags these as redundant self calls, which makes sense to me because they're nested inside another `self` whereas the others aren't. If you want to poke around further and find out if we can remove the other `self` references I certainly won't stop you.

The commit message is a vague attempt at humour. Introspection is essentially the examination of self, so:
```
pathname: become less introspective
```
Is more or less an attempt at shorthand humour for:
```
pathname: prune redundant self calls
```
Just a minor attempt to make things less extremely tedious.